Верстка страниц на html

VladislavShtefan opened the pull request:
Css только начал

VladislavShtefan committed:
ДЗ. Верстка главной страницы

VladislavShtefan committed:
ДЗ исправил

VladislavShtefan committed:
ДЗ сделал

Полностью переписал главную страницу, написал классы, вставил картинки, вставил ссылки в некоторых кнопках и логотипе. И просто привел код в порядок. Посмотри пожалуйста. Спасибо!

VladislavShtefan committed:
Главная ДЗ

Правки +

VladislavShtefan committed:
Все странички на html с классами

Надеюсь я правильно все увидел в макете.  К css вот только приступил.

VladislavShtefan committed:
Главная со стилями

Пока получается не особо

VladislavShtefan committed:
Странички на HTML и стили

VladislavShtefan committed:
Остальные странички на html без стилизации

VladislavShtefan committed:
Проблемы с баннером, и подвалом сайта

Пока что, я не понимаю что надо переделать.

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,397 @@
+    <!-- footer -->
+    <footer class="footer">
+        
+            <section class="footer_nav">
```
Ты можешь сделать на три списка одну секцию и уже ей задать флекс

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,397 @@
+            </div>
+        </section>
+
+            <img src="img/long_line.png" alt="#">
```
Картинкой полоску можешь не делать, а на секцию с тремя списками прописать border-bottom

VladislavShtefan committed:
Главная страница

Проблемы с "подвалом" и карточками "о нас"

VladislavShtefan committed:
Работа с первым модулем. 

Не уверен в том, что написал все как нужно, но по макету вроде все начинает сходиться.

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,242 @@
+
+<!-- HEADER -->
+        <header class="header__container container">
+            <ul class="header__list-one">
```
Зачем тут модификатор one?

VladislavShtefan committed:
Стрелки переключения 

не получается сделать стрелки на весь экран

VladislavShtefan committed:
Стилизация формы заявки

VladislavShtefan committed:
Сайт к 16.03

Не получилось закинуть файлы через desktop. На сайте нет стилизации ползунка, эффекты при наведении на элементы (кнопки и т.д.), так же расставил focus для клавиатуры. Пока скинул сюда, но продолжу делать.

VladislavShtefan committed:
Дополнение к сайту на 16.03

Забыл скинуть

VladislavShtefan committed:
1 Блок. Наконец все отправилось на гит

Вроде должно было сработать

VladislavShtefan committed:
Учебный сайт (повторно)

Залил все новые файлы. Вроде все заработало.
